src/irmin-pack/unix/ext.ml

```diff
@@ -417,7 +420,7 @@ module Maker (Config : Conf.S) = struct
                   let result =
                     let open Result_syntax in
                     match (status, gc_output) with
-                    | Lwt_unix.WEXITED 0, Ok copy_end_offset ->
+                    | Lwt_unix.WSIGNALED _, Ok copy_end_offset ->
```

Comment:
```suggestion
                    | Lwt_unix.WSIGNALED 9, Ok copy_end_offset ->
```
Does this work?

Reply:
Let's unlink the result file before forking. To make sure that we are not catching the result file from a previously aborted GC

Reply:
> `| Lwt_unix.WSIGNALED 9,  `
> Does this work?

No, somehow it becomes a (-7) when catched here. I opted for `Sys.sigkill` instead. (changed a bit the structure as the match on  `Lwt_unix.WSIGNALED  Sys.sigkill` does not work...)

Reply:
Nevermind, leave `_`